app/package.json

```diff
@@ -1,8 +1,8 @@
 {
   "private": true,
   "scripts": {
-    "lint": "eslint --max-warnings=0 .",
-    "postinstall": "patch-package"
+    "build": "yarn patch-package",
```

Comment:
I think yarn supports patching dependencies out-of-the-box; let's migrate from `patch-package` to `yarn patch` but in a separate PR.